### PR TITLE
W3HW5 model card for RandomForest classifiers

### PR DESCRIPTION
**Pull request by yevhen-k**
Hey @truskovskiyk 

Here is model card for the RandomForest. Thank you for the tool, will use it :)
Please, check it

### CODE REVIEW

File: week-3/model-card.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        mctlib.Graphic(image=roc_curve),
+        mctlib.Graphic(image=confusion_matrix)
+    ]
+    mct.update_model_card(model_card)
+
+    ### Generate model card
+    # Return the model card document as an HTML page
+    html = mct.export_format()
+    with open("card.html", "wt") as f:
+        f.write(html)
```
Comment (truskovskiyk) [inline week-3/model-card.py]: should we make this a function?

File: week-3/model-card.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    model_card.quantitative_analysis.graphics.collection = [
+        mctlib.Graphic(image=roc_curve),
+        mctlib.Graphic(image=confusion_matrix)
+    ]
+    mct.update_model_card(model_card)
+
+    ### Generate model card
+    # Return the model card document as an HTML page
+    html = mct.export_format()
+    with open("card.html", "wt") as f:
```
Comment (truskovskiyk) [inline week-3/model-card.py]: please remove hardcode

File: week-3/model-card.py [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    model_card.model_details.overview = (
+        'Simple Card With RandomForest Classifier for Dummy Data')
+    model_card.model_details.owners = [
+        mctlib.Owner(name= 'Yevhen K.', contact='name@example.com')
+    ]
+    model_card.model_details.references = [
+        mctlib.Reference(reference='https://www.tensorflow.org/responsible_ai/model_card_toolkit/examples/Scikit_Learn_Model_Card_Toolkit_Demo'),
+        mctlib.Reference(reference='https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html')
+    ]
+    model_card.model_details.version.name = str(uuid.uuid4())
```
Comment (truskovskiyk) [inline week-3/model-card.py]: also might be another function

File: week-3/model-card.py (hunk 1 above)
Comment (yevhen-k) [reply week-3/model-card.py]: code is updated

File: week-3/model-card.py (hunk 2 above)
Comment (yevhen-k) [reply week-3/model-card.py]: hardcoded filename removed

File: week-3/model-card.py (hunk 3 above)
Comment (yevhen-k) [reply week-3/model-card.py]: some refactoring is done

**Review by truskovskiyk — APPROVED** [review]
thank!

**Pull request merged**